components/MainHeader/MainHeader.tsx

```diff
@@ -2,33 +2,17 @@ import Link from 'next/link';
 
 import styles from './mainHeader.module.scss';
 
-const links = [
-  { label: 'Aktualności', href: '/aktualnosci', openInNewWindow: false, icon: 'inbox' },
-  { label: 'O serwisie', href: '/o-serwisie', openInNewWindow: false, icon: 'question-circle' },
-  { label: 'Zgłoś uwagi', href: '/zglos-uwagi', openInNewWindow: false, icon: 'comment' },
-  {
-    label: 'Facebook',
-    href: 'https://facebook.com/polskifrontend',
-    openInNewWindow: true,
-    icon: 'facebook',
-  },
-];
-
 export const MainHeader = () => {
   return (
-    <header>
-      <ul className={styles.headerList}>
-        {links.map(({ label, href, openInNewWindow, icon }) => (
-          <li className={styles.headerItem} key={href}>
-            <Link href={href}>
-              <a {...(openInNewWindow && { target: '_blank' })} title={label}>
-                <span className={`icon- ${styles.icon}`}>{icon}</span>
-                <span className={styles.label}>{label}</span>
-              </a>
-            </Link>
-          </li>
-        ))}
-      </ul>
-    </header>
+    <section className={styles.headerSection}>
+      <Link href="/">
+        <a title="Przejdź na stronę główną">
+          <h1 className={styles.title}>
+            <span className="sr-only">Polski front-end</span>
+            <img src="/logo.svg" alt="Logo polskifrontend" />
```

Comment:
Skoro wyżej jest już nazwa, to mam wrażenie że alt do obrazka jest zbędny, i lepiej byłoby zostawić alt=""